### ISSUE TITLE
Qt6: WARNING: skipping QEventPoint when using trackpad (maybe macOS only?)

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/napari#5530]
## 🐛 Bug

When using Qt6 backend, while mousing using a trackpad, such as adding Points to a Points layer, a warning is generated, occasionally/randomly:
```
WARNING: skipping QEventPoint(id=1 ts=0 pos=0,0 scn=873.796,480.551 gbl=873.796,480.551 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-873.796,-480.551 last=-873.796,-480.551 Δ 873.796,480.551) : no target window
12:40:45 WARNING skipping QEventPoint(id=1 ts=0 pos=0,0 scn=873.796,480.551 gbl=873.796,480.551 Released ellipse=(1x1 ∡ 0) vel=0,0 press=-873.796,-480.551 last=-873.796,-480.551 Δ 873.796,480.551) : no target window
```
First reported in the Qt6 tracking issue:
https://github.com/napari/napari/issues/3880#issuecomment-1000927698

This occurs on both PyQt6 and PySide6, it seems to be related to an upstream Qt6 issue:
https://bugreports.qt.io/browse/QTBUG-103935?gerritReviewStatus=All

It may be a macOS only issue or a trackpad only issue and it appears that the warning is harmless (see the qt.io thread). The suggested solution is to unsetting `WA_AcceptTouchEvents`, which is upstream of napari in vispy.

## To Reproduce

On a computer using a trackpad, such as mac laptop:

1. install Qt6 backend (pyqt6 or pyside6)
2. `pip install napari` (no [all] to use the Qt6 backend)
3. open Astronaut sample image
4. make a labels or points layer — this reliably triggers the warning at least once.

This can also be reproduced with just vispy using for example the `turntable_box.py` example and mousing around.

## Expected behavior

No warning, even if it's harmless.

## Environment

 Qt6 (pyqt6 or pyside6), macOS 13.2 arm64, laptop, using trackpad.

## Additional context

This needs to be fixed upstream in vispy. 
The key code is here:
https://github.com/vispy/vispy/blob/c0d7457a7b1717d47baa8b8622faa2c67bbe4be0/vispy/app/backends/_qt.py#L413-L423

Based on the Qt thread linked above and docs, TouchEvents are intended for touchscreens, while gestures should be handled using QtNativeGestureEvent:
https://doc.qt.io/qtforpython/PySide6/QtGui/QNativeGestureEvent.html#more

**Comment by psobolewskiPhD** [comment]
Using pyqt6 (6.5.0) and vispy-0.12.3.dev44 this is resolved thanks to @aganders3 https://github.com/vispy/vispy/issues/2455 

So next release of vispy that's <13.0 will solve this!

FYI Just for anyone reading vispy PRs carefully: pinch-to-zoom works in vispy, but not yet in napari.

**Comment by psobolewskiPhD** [comment]
The fix in vispy was released in 0.13, so once we move our pin, this issue will be closable.

**Comment by psobolewskiPhD** [comment]
Closed by https://github.com/napari/napari/pull/6025 which has been merged.
